crowbar_framework/lib/crowbar/repository.rb

```diff
@@ -256,6 +256,14 @@ def provided_with_enabled(feature,
             unless repos[:inactive][r.arch.to_sym].include?(r.name)
               repos[:inactive][r.arch.to_sym].push(r.name)
             end
+
+            next if r.required == "mandatory"
+              repos[:optional] ||= {}
```

Comment:
the indentations look somehow wrong here

Reply:
True